package.json

```diff
@@ -70,8 +69,9 @@
     "mocha": "^10.2.0",
     "prettier": "^3.1.0",
     "semantic-release": "^24.0.0",
-    "typescript": "^5.4.1",
-    "ts-node": "^10.9.1"
+    "sinon": "^18.0.0",
```

Comment:
Why was this package introduced?

Reply:
not sure, I think it was auto installed for some reason. gonna revert the change